airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/streams/streams.py

```diff
@@ -149,6 +149,58 @@ class AdAccount(FBMarketingStream):
     use_batch = False
     enable_deleted = False
 
+    def createArraySchema(self, array_val):
```

Comment:
should this log something if nothing is returned?

Reply:
It will return empty json